documentation/modules/rn-2.3.adoc

```diff
@@ -71,3 +71,9 @@ The error status message for a VM with no operating system on the *Migration pla
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.{rhv-short} destination VMs fail to boot if source keys are encrypted with TPM
+{project-short} does not use `virt-v2v` when it performs migrations from {rhv-short}. As a result any data in the source VM that is encrypted with TPM is not migrated to destination VMs. Therefore, any source keys encrypted with TPM are not migrated. This makes it impossible to boot destination VMs.
```

Comment:
this part would also be relevant as from 2.4, currently (2.3.4) we invoke virt-v2v for both vsphere and rhv - need to drop that sentense